lit/docs/guides/container-images.lit

```diff
@@ -0,0 +1,318 @@
+      branch: master
+
+  # Where we will push the image to
+  - name: simple-image
+    type: registry-image
+    icon: docker
+    source:
+      repository: ((image-repo-name))/simple-image
+      username: ((registry-username))
+      password: ((registry-password))
+  }}
+
+  \inline-header{Create the Job}
+
+  Next we will create a \reference{jobs}{job} that will build and push our container image.
+
+  To build the job we will need to pull in the repo where the \code{Dockerfile} is.
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources: ... # omitting resource section from above
+
+  jobs:
+  - name: build-and-push
+    plan:
+    - get: concourse-examples
+  }}
+
+  \inline-header{Build the Image}
+
+  The second step in our job will build the container image.
+
+  To build the container image we are going to use the
+  \link{oci-build-task}{https://github.com/vito/oci-build-task}. The
+  \link{oci-build-task}{https://github.com/vito/oci-build-task} is a container
+  image that is meant to be used in a Concourse \reference{tasks}{task} to build other
+  container images. Check out the
+  \link{README}{https://github.com/vito/oci-build-task/blob/master/README.md} in
+  the repo for more details on how to configure and use the oci-build-task in
+  more complex build scenarios.
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources: ... # omitting resource section from above
+
+  jobs:
+  - name: build-and-push
+    plan:
+    - get: concourse-examples
+    - task: build-image
+      privileged: true # oci-build-task must run in a privileged container
+      config:
+        platform: linux
+        image_resource:
+          type: registry-image
+          source:
+            repository: vito/oci-build-task
+  }}
+  \inset{} {- spacer -}
+
+  Next we will add
+  \link{concourse-examples}{https://github.com/concourse/examples} as an
+  \reference{schema.task.inputs}{input} to the build task to ensure the
+  artifact from the \reference{get-step}{get step} (where our \code{Dockerfile}
+  is fetched) is mounted in our \code{build-image} step.
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources: ... # omitting resource section from above
+
+  jobs:
+  - name: build-and-push
+   plan:
+   - get: concourse-examples
+   - task: build-image
+     privileged: true # oci-build-task must run in a privileged container
+     config:
+       platform: linux
+       image_resource:
+         type: registry-image
+         source:
+           repository: vito/oci-build-task
+       inputs:
+       - name: concourse-examples
+  }}
+  \inset{} {- spacer -}
+
+  The oci-build-task \link{outputs the built container
+  image}{https://github.com/vito/oci-build-task#outputs} in a directory called
+  \code{image}. Let's add image as an output of our task so we can publish it
+  in a later step.
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources: ... # omitting resource section from above
+
+  jobs:
+  - name: build-and-push
+   plan:
```

Comment:
nit: This is one space unaligned! Not sure if you want to bother fixing all of them, theres a few more places where it is one space off haha